functions/src/api/authenticateJWT.ts

```diff
@@ -0,0 +1,46 @@
+import * as jsonWebToken from 'jsonwebtoken'
+import { Request, Response } from 'express'
+import { VerifyErrors } from 'jsonwebtoken'
+import { JWTData } from '../types/JWTData'
+import { JWTToken } from '../types/JWTToken'
+import { isTokenValidInDb } from '../db/isTokenValidInDb'
+import { assertJWTEnv } from '../utils/assertConfig'
+
+export const authenticateJWT = (
+    req: Request,
+    res: Response,
+    next: Function
+) => {
+    const jwtKey = assertJWTEnv()
```

Comment:
I would use `getJWTKey` here, since:
- you're expecting a value as a return
- for consistency: the variable you affect the result of the function to is called `jwtKey`

The `get` method can still raise an error if the instance is not found. But `assert` is normally used in tests: as a reader, I would not expect an `assert` to return a value

Reply:
changed! As a side not, android framework do use assert at runtime to check the input on a method.

Reply:
could u pls provide an example link for my personal improvement ? :)

Reply:
well, I believe to had read some in the past but was unable to find example now. My assumption is that it's code generated from annotation that produce this, like @NonNull .